website/content/en/docs/getting-started/users.md

```diff
@@ -54,19 +54,19 @@ To set up ClusterLink on a Kubernetes cluster, follow these steps:
 1. {{< anchor install-cl-operator >}}Install ClusterLink deployment operator:
 
    ```sh
-   clusterlink deploy peer --autostart --name <peer_name> --fabric <fabric_name>
+   clusterlink deploy peer --name <peer_name> --fabric <fabric_name>
    ```
 
    This command will deploy the ClusterLink operator on the `clusterlink-operator` namespace
     and convert the peer certificates to secrets in the namespace where ClusterLink components will be installed.
     By default, the `clusterlink-system` namespace is used.
+    in addition it  will create a ClusterLink K8s custom resource object and deploy it to the operator.
```

Comment:
```suggestion
    in addition it will create a ClusterLink K8s custom resource object and deploy it to the operator.
```
I also think the CR type is `Instance`, not `ClusterLink`. Might be best to fully qualify it?